pkg/wharfapi/token.go

```diff
@@ -66,15 +67,39 @@ func (c Client) GetToken(token string, userName string) (Token, error) {
 	return tokens[0], nil
 }
 
+func (c Client) PutToken(token Token) (Token, error) {
+	body, err := json.Marshal(token)
+	if err != nil {
+		return Token{}, err
+	}
+
+	log.WithField("project", string(body)).Traceln()
```

Comment:
Is this token sensitive? Maybe it shouldn't be logged.

Reply:
Yes i belive so. Im unsure about the logging strategy here though. We dont really log anything from this repo. It appears the trace level of the log never gets output. 
With the descriptions available on the http responses i belive this logging wouldnt do much more good in its current state. I cant think of any more info we would really need/want here so i would advocate for dropping this traceln log from the put method here and in the provider.